code/player/TTTPlayer.Commands.Server.cs

```diff
@@ -79,6 +79,11 @@ public static void RespawnPlayer(int id)
         [ServerCmd(Name = "ttt_requestitem")]
         public static void RequestItem(string itemName)
         {
+            if (itemName == null)
```

Comment:
I saw you used `is null` earlier, we should probably just stick to one version.

Reply:
Should be done in another refactoring, after we got to a final decision.